packages/theme/src/fonts/index.ts

```diff
@@ -1,5 +1,21 @@
+/**
+ * Fonts
+ *
+ * {@link https://zeroheight.com/3ddd0f892/p/211297-typography/b/655ccf}
+ */
 export const primary = {
   family: 'Lato, sans-serif',
+  variant: {
+    p1: '2rem',
+    p2: '1.8rem',
+    p3: '1.6rem',
+    p4: '1.4rem',
+    caption1: '1.2rem',
+    button1: '2rem',
+    button2: '1.8rem',
+    button3: '1.6rem',
+    button4: '1.4rem',
+  },
```

Comment:
Could we make the variant match the element names? E.g., `h1`, `h2`, `button`, etc.

Reply:
Updated!